<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1610</width>
    <height>896</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_42">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="title">
            <string>MVC (N-m)</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="lbl_playermvc">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="title">
            <string>DAQ Port</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QListWidget" name="list_comport">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_refreshcomlist">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Refresh List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_selectcomport">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_comport">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_daqportstatus">
              <property name="font">
               <font>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btn_resetserial">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Reset Serial</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="title">
            <string>Player</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="lbl_playername">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Target Game</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_30">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Player</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineedit_newplayer">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="placeholderText">
                     <string>Type New Player Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_3">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>14</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>New Player</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="4">
                   <widget class="QPushButton" name="btn_addplayer">
                    <property name="minimumSize">
                     <size>
                      <width>75</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>75</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="label_4">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>23</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Returning Player</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QCheckBox" name="checkbox_isadult">
                    <property name="text">
                     <string>Adult</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1" colspan="4">
                   <widget class="QComboBox" name="combo_returningplayer">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="currentText">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_8">
                 <property name="title">
                  <string>MVC</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QPushButton" name="btn_recordmvc">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Record</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="btn_analyzemvc">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Analyze</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_9">
                 <property name="title">
                  <string>Target Game</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="1" column="0">
                   <widget class="QLabel" name="label">
                    <property name="font">
                     <font>
                      <pointsize>17</pointsize>
                     </font>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="text">
                     <string>Targets Hit: </string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="lbl_targethitcount">
                    <property name="font">
                     <font>
                      <pointsize>17</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="btn_targetgamestop">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Stop</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="btn_targetgamestart">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Start</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="pbar_targetholdprog">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="textVisible">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="lbl_livenotes">
                 <property name="font">
                  <font>
                   <pointsize>17</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_16">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Leaderboard</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="rbtn_youthleaderboard">
                    <property name="text">
                     <string>Youth</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QRadioButton" name="rbtn_adultleaderboard">
                    <property name="text">
                     <string>Adult</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QRadioButton" name="rbtn_overallleaderboard">
                    <property name="text">
                     <string>Overall</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="5">
                   <widget class="QTableWidget" name="table_targetleaderboard"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QVBoxLayout" name="layout_pyqtgraph"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>125</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
